tests/services/inventory/test_data/additional_cai_resources.dump

```diff
@@ -60,8 +60,22 @@
+# Kubernetes ClusterRoleBinding (id 926x)
+{"name":"//container.googleapis.com/projects/1042/zones/us-central1-b/clusters/test-cluster/k8s/rbac.authorization.k8s.io/clusterrolebindings/9261","asset_type":"rbac.authorization.k8s.io/ClusterRoleBinding","resource":{"version":"v1","discovery_document_uri":"https://raw.githubusercontent.com/kubernetes/kubernetes/master/api/openapi-spec/swagger.json","discovery_name":"io.k8s.api.rbac.v1.ClusterRoleBinding","data":{"id":"9261","metadata":{"annotations":{"rbac.authorization.kubernetes.io/autoupdate":"true"},"creationTimestamp":"2017-08-09T23:59:47Z","labels":{"kubernetes.io/bootstrapping":"rbac-defaults"},"name":"cluster-admin","resourceVersion":"30377996","selfLink":"/apis/rbac.authorization.k8s.io/v1/clusterrolebindings/cluster-admin","uid":"123"},"roleRef":{"apiGroup":"rbac.authorization.k8s.io","kind":"ClusterRole","name":"cluster-admin"},"subjects":[{"apiGroup":"rbac.authorization.k8s.io","kind":"Group","name":"system:masters"}]}}}
 # PubSub Subscription
-{"name":"//pubsub.googleapis.com/projects/project2/subscriptions/test-subscription","asset_type":"pubsub.googleapis.com/Subscription","resource":{"version":"v1","discovery_document_uri":"https://pubsub.googleapis.com/$discovery/rest","discovery_name":"Subscription","parent":"//cloudresourcemanager.googleapis.com/projects/1042","data":{"ackDeadlineSeconds":10,"expirationPolicy":{"ttl":"2678400s"},"messageRetentionDuration":"604800s","name":"projects/project2/subscriptions/test-subscription","topic":"projects/project2/topics/test-topic0123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123"}}}
+{"name":"//container.googleapis.com/projects/1042/zones/us-central1-b/clusters/test-cluster/k8s/rbac.authorization.k8s.io/clusterrolebindings/9261","asset_type":"rbac.authorization.k8s.io/ClusterRoleBinding","resource":{"version":"v1","discovery_document_uri":"https://raw.githubusercontent.com/kubernetes/kubernetes/master/api/openapi-spec/swagger.json","discovery_name":"io.k8s.api.rbac.v1.ClusterRoleBinding","data":{"id":"9261","metadata":{"annotations":{"rbac.authorization.kubernetes.io/autoupdate":"true"},"creationTimestamp":"2017-08-09T23:59:47Z","labels":{"kubernetes.io/bootstrapping":"rbac-defaults"},"name":"cluster-admin","resourceVersion":"30377996","selfLink":"/apis/rbac.authorization.k8s.io/v1/clusterrolebindings/cluster-admin","uid":"9261"},"roleRef":{"apiGroup":"rbac.authorization.k8s.io","kind":"ClusterRole","name":"cluster-admin"},"subjects":[{"apiGroup":"rbac.authorization.k8s.io","kind":"Group","name":"system:masters"}]}}}
```

Comment:
It looks like the pubsub subscription sample got overwritten, can you check?

Reply:
Reverted accidental changes.